# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit-test/tests/explicit-resource-management/disposable-stack-adopt-dispose.js
# Commit: ce53cf8941e8
# Full Hash: ce53cf8941e861b97598f4c250daab63727ae1d2
# Author: Debadree Chatterjee <debadree333@gmail.com>
# Date: 2024-11-03 21:45:44
# Regressor Bug: 1927195
# File Overlap Count: 1
# Description:
#   Bug 1927195 - Part 2: Pass shell option in explicit resource management jit-tests. r=arai
#   
#   Depends on D226991
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D226992
# ==============================================================================

diff -r e33308b4d926 -r ce53cf8941e8 js/src/jit-test/tests/explicit-resource-management/disposable-stack-adopt-dispose.js
--- a/js/src/jit-test/tests/explicit-resource-management/disposable-stack-adopt-dispose.js	Sun Nov 03 08:32:25 2024 +0000
+++ b/js/src/jit-test/tests/explicit-resource-management/disposable-stack-adopt-dispose.js	Sun Nov 03 08:32:26 2024 +0000
@@ -1,4 +1,4 @@
-// |jit-test| skip-if: !getBuildConfiguration("explicit-resource-management")
+// |jit-test| skip-if: !getBuildConfiguration("explicit-resource-management"); --enable-explicit-resource-management
 
 load(libdir + "asserts.js");
 